Python_DataEngineering/intern.py

```diff
@@ -0,0 +1,47 @@
+import urllib
+import requests
+import json
+import mysql.connector
+from mysql.connector import Error
+
+DB_NAME = 'BikePoints'
+cnx = mysql.connector.connect(user='root',password='1234')
+cursor = cnx.cursor()
+
+cursor.execute("USE {}".format(DB_NAME))
+
+data = requests.get('https://api.tfl.gov.uk/bikepoint')
+bike_data=data.json()
+
+# print(type(bike_data))
+
+for bd in bike_data:
+    id = int(bd['id'].split('_')[-1])
+    name = str(bd['commonName'])
+    aP = bd['additionalProperties']
+    installed = aP[1]['value']
```

Comment:
You're assuming that the format in which the bike data comes will be constant (1 for installed, 2 for locked, etc), what if this changes? Can you make it more generic so that even if the order is changed, your code will still hold good?

Reply:
That will be problem. I changed the code so that it will work even if the order is changed.